cryptlex/lexfloatclient/lexfloatclient.py

```diff
@@ -19,6 +20,11 @@ def __init__(self, name, enabled, data):
         self.enabled = enabled
         self.data = data
 
+class HostConfig(object):
+    def __init__(self, maxOfflineLeaseDuration):
+        self.maxOfflineLeaseDuration = maxOfflineLeaseDuration
+
```

Comment:
@muneebkq In Python, the snake case is commonly followed. Please update the above.